docs/sdks/server-sdks/python.md

```diff
@@ -209,7 +223,7 @@ But that’s not all: the function also takes an optional input for entity prope
 
 ### A. Optional Properties for Targeting 
 
-Most entities on which we run feature flags have properties: sessions have browser types, users have loyalty status, corporate clients have a number of employees, videos have close-caption available or not, sport teams have a league, etc. If you want to decide how a feature flag behaves, or whether an experiment is ran on a certain entity based on those, you need to send that information too. When assigning entities, you can pass that additional information through `subject_attributes`: an optional dictionary that details entity properties. 
+Most entities on which we run feature flags have properties: sessions have browser types, users have loyalty status, corporate clients have a number of employees, videos have close-caption available or not, sport teams have a league, etc. If you want to decide how a feature flag behaves, or whether an experiment is run on a certain entity based on those, you need to send that information too. When assigning entities, you can pass that additional information through `subject_attributes`: an optional dictionary that details entity properties. 
```

Comment:
The past participle of "run" is "run" rather than "ran"

Reply:
I’ve long abandoned the expectation that I would know those without Grammarly telling me off about those. It didn’t work in the GitHub environment and that genuinely scared me.

Reply:
> I’ve long abandoned the expectation that I would know those without Grammarly telling me off about those. It didn’t work in the GitHub environment and that genuinely scared me.

Same! I legitimately spent 15 minutes yesterday researching how to integrate Grammarly with markdown. Sadly I didn't see any good solutions that wouldn't require me to use a different IDE. I'm team JetBrains for life. Fortunately, it does have a spelling/grammar checker which helped me catch some of these issues, but it's nowhere near as advanced as Grammarly